hokusai/production.yml

```diff
@@ -29,8 +29,10 @@ spec:
         - containerPort: 3000
         resources:
           requests:
-            cpu: "0.7"
+            cpu: "0.5"
+            memory: "512Mi"
           limits:
+            cpu: "1"
```

Comment:
Since a single NodeJS process can't multithread it should at most consume 1 CPU.